src/locales/en.json

```
@@ -15,6 +15,7 @@
     "label": "Data source"
   },
   "trips": {
+    "trips": "Trips",
```

nit: Peut-être renommer la clé parente ?
Ou ?
```suggestion
    "self": "Trips",
```

---

mouaif... hmmmmmoooaaarrfff... 🤔 je comprends l'idée du self mais la trad n'a rien à voir avec la clé... je trouve pas fou fou non plus le trips.trips...